tests/.test.env

```diff
@@ -1,7 +1,10 @@
 # Test configuration variables
 
-DAILY_READ_ORDER_PORTAL_URL="https://orderportal.example.com"
+DAILY_READ_ORDER_PORTAL_URL="https://orderportal.example.com/"
+DAILY_READ_ORDER_PORTAL_API_KEY='fake12349183o18390301'
 DAILY_READ_DATA_LOCATION = "tests/test_data_location"
+DAILY_READ_REPORTS_LOCATION='tests/test_reports_location'
 DAILY_READ_FETCH_FROM_NGIS = true
 DAILY_READ_FETCH_FROM_SNPSEQ = true
 DAILY_READ_FETCH_FROM_UGC = true
+#DAILY_READ_LOG_LOCATION='tests/log_location'
```

Comment:
You want to keep this comment here?

Reply:
Maybe as a reminder for future tests? I can get rid of it too.